ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,53 @@
-export function generateMetadata(): string {
+import { LoopPermissions } from './loop-permissions';
+
+function isPermissionDefined(permission: string[]): boolean {
+  if (permission.length === 0 || 
+      permission === undefined) {
+    return true;
+  } 
+  return false;
+}
+
+function buildUrlPermissions(permissions: LoopPermissions): string {
+  if (isPermissionDefined(permissions.urlPermissions)) {
```

Comment:
I don't think we should throw an error if this value is not defined; suppose they just don't want to work with network services?

Reply:
I agree 100% for aptitudes. How do you feel about other permissions @richseviora ?